CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -811,6 +812,12 @@ public BitmapInfo CreateBitmapInfo(bool isPopup)
             return BitmapFactory.CreateBitmap(isPopup);
         }
 
+        float _deviceScaleFactor = 1.0F;
```

Comment:
Underscores only in `c++` code - consistency.
Variables should also be declared in the one place for consistency.